### DISCUSSION TITLE
When to Use Static Generation v.s. Server-side Rendering

### DISCUSSION THREAD

**Discussion by ghost** [mrhappyma/site#2] [Blog Comments]
# When to Use Static Generation v.s. Server-side Rendering

http://localhost:3000/posts/sgs-ssr

**Comment by mrhappyma** [comment]
Hey look, comments are working!

**Comment by mrhappyma** [comment]
| Header | Header | Header | Header |
|--------|--------|--------|--------|
| Cell | Cell | Cell | Cell |
| Cell | Cell | Cell | Cell |
| Cell | Cell | Cell | Cell |
| Cell | Cell | Cell | Cell |
| Cell | Cell | Cell | Cell |
